dist/robotframework/res/riot_base.keywords.txt

```diff
@@ -7,10 +7,10 @@ Library     Collections
 *** Variables ***
 ${RIOTTOOLS}            %{RIOTBASE}/dist/tools
 
+
 *** Keywords ***
-Reset Application
+RIOT Reset
     [Documentation]     Reset the test application
-    [Arguments]         ${sleep_before}=3  ${sleep_after}=3
-    Sleep               ${sleep_before}
+    ${sleep_after}=     Get Environment Variable  RESET_WAIT  default=3
```

Comment:
Note this allows to overwrite the sleep after reset with a local (board specific) wait time and defaults to 3 seconds. This is used for instance for the Arduino to extend the wait for its slow boot.